.github/workflows/release-please.yml

```diff
@@ -54,11 +54,15 @@ jobs:
 
       # get main again
       - name: Checkout Repository
+        # only checkout if a release has been created
+        if: ${{ steps.release.outputs.releases_created }}
         uses: actions/checkout@v4
         with:
           fetch-depth: 0
 
       - name: Rebuild Packages
+        # only checkout if a release has been created
```

Comment:
nit: update the comment copied from the previous step or remove it

Reply:
I applied the suggestion via a comment

Reply:
see https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2160/commits/2acfea62a2075580c2535a3567b56a5e85c2ccb5 and https://github.com/open-telemetry/opentelemetry-js-contrib/pull/2160/commits/fe6da36260685c499a5073dc2c205d713768b622 :slightly_smiling_face:

Reply:
Oops good eye thanks both!